docs/user-guide/cli-usingcli.md

```diff
@@ -94,24 +385,102 @@ To enable the feature, set an environment variable named `ZOWE_OPT_PASSWORD` wit
 
 **Tip** The procedure for setting environment variables is dependent on your operating systems. Refer to documentation for your OS to learn about setting environment variables.
 
-### Change the keyword for prompt
+#### Change the keyword for prompt
 
 The default keyword that enables prompting in Zowe CLI is `"PROMPT*"`. You might want to change the keyword if there is a chance that `"PROMPT*"` could exist as a valid value for the field. For example, if you mask the `data-set` argument and are working with real mainframe data sets that begin with the characters `"PROMPT*"`.
 
 To configure the keyword, choose a new value. Then define the value to to the environment variable on your computer named `ZOWE_PROMPT_PHRASE`.
 
-## Zowe CLI command groups
-Zowe CLI contains command groups that focus on specific business processes. For example, the `zos-files` command group
-provides the ability to interact with mainframe data sets. This article provides you with a brief synopsis of the tasks that you can perform with each group. For more information, see [Display Zowe CLI Help](#displaying-zowe-cli-help).
+## Writing scripts
 
-The commands available in the product are organized in a hierarchical structure. Command groups (for example, `zos-files`) contain actions (for example, `create`) that let you perform actions on specific objects (for example, a specific type of data set). For each action that you perform on an object, you can specify options that affect the operation of the command.
+You can combine multiple Zowe CLI commands in bash or shell scripts to automate actions on z/OS. You can implement scripts to enhance your development workflow, automate repetitive test or build tasks, and orchestrate mainframe actions from continuous integration/continuous deployment (CI/CD) tools such as Jenkins or TravisCI.
+
+- [Example: Clean up Temporary Data Sets](#exampleOne)
+- [Example: Submit Jobs and Save Spool Output](#exampleTwo)
+
+**Note:** The type of script that you write depends on the programming languages that you use and the environment where the script is executed. The following procedure is a general guide to Zowe CLI scripts, but you might need to refer to third-party documentation to learn more about scripting in general.
+
+**Follow these steps:**
+
+1. Create a new file on your computer with the extension .sh. For example, `testScript.sh`.
+
+    **Note:** On Linux, an extension is not required. You make the file executable by issuing the command `chmod u+x testScript`.
+
+2. At the top of the file, specify the interpreter that your script requires. For example, type `#!/bin/sh` or `#!/bin/sh`.
```

Comment:
Windows doesn't support shebang lines (it uses file extensions to determine the correct interpreter to execute). So step 2 is required only for Linux or macOS, same as step 1a.